assets/js/mixins/formatter.js

```diff
@@ -14,9 +14,17 @@ const ENERGY_PRICE_IN_SUBUNIT = {
   USD: "¢", // US cent
 };
 
+export const WATT_FORMAT = Object.freeze({
+  WATT: "W",
+  KW: "kW",
+  MW: "MW",
+  AUTO: "",
+});
+
 export default {
   data: function () {
     return {
+      WATT_FORMAT: WATT_FORMAT,
```

Comment:
Is this used?

Reply:
That's what I meant with
> I was unsure how I should export the enum.

Removing it triggers this error in the developer console: `Uncaught (in promise) TypeError: _ctx.POWER_UNIT is undefined`

Reply:
Ah ok, now I see. We have a couple of cases where we call `fmtWh` directly from `<template>`. Therefore, we need this in component scope.

But we could use the short syntax: `WATT_FORMAT,`